tests/conftest.py

```diff
@@ -101,6 +102,64 @@ def get_testbed_type(self, topo_name):
             tb_type = 't0'
         return tb_type
 
+    def _parse_dut_port_index(self, port):
+        """
+        parse port string
+
+        port format : dut_index.port_index@ptf_index
+
+        """
+        m = re.match("(\d+)\.(\d+)@(\d+)", port)
+        (dut_index, port_index, ptf_index) = (int(m.group(1)), int(m.group(2)), int(m.group(3)))
+
+        return (dut_index, port_index, ptf_index)
+
+    def calculate_ptf_index_map(self, line):
+        map = defaultdict()
+        if len(line['duts']) <= 1:
```

Comment:
we may want to move to the multi-duts format in topology file in the future.

Reply:
I think we can move generic information to topology file. But here line['duts'] are defining the specific information of the testbed, it provides list of hostnames for the duts in this testbed. This part probably have to stay in the testbed definition.

Reply:
Did you mean ignore this field and use topology property 'dut_num' for this test instead?

Reply:
Per our discussion offline. Removed the dut count check.